kafka/ci/resources/kafka.yaml

```diff
@@ -305,8 +305,8 @@ init_config:
         bean: 'kafka.server:type=KafkaRequestHandlerPool,name=RequestHandlerAvgIdlePercent'
         attribute:
           Count:
-            metric_type: rate
-            alias: kafka.request.handler.avg.idle.pct.rate
+            metric_type: gauge
```

Comment:
@truthbk Thank you for taking a look.  The issue with the metric as it currently stands is that it's not a typical meter.  Instead of counting events, [it's counting nanoseconds of idle time](https://github.com/apache/kafka/blob/cb674e5487f3f56647546b323dfe4fd45ccf0186/core/src/main/scala/kafka/server/KafkaRequestHandler.scala#L47-L54) so that if you ask for the rate in units of nanoseconds, you get a value between 0.0 and 1.0.  The `Count` attribute which is current being queried represents nanoseconds of idle time but it's being declared as a `rate` which assume events/sec in DataDog (I think).  So whatever is being shown in the current dashboards will be idle nanoseconds/seconds which should be 10**9 time too big??

![screen shot 2017-04-12 at 3 12 37 pm](https://cloud.githubusercontent.com/assets/74996/24993202/374ca86e-1fda-11e7-86d4-e2b732e5a3a1.png)

That said, we could leave both in there.  I'll update the PR.